TradeItIosEmsApi/TradeItConnector.m

```diff
@@ -433,6 +421,8 @@ - (void)sendEMSRequestReturnJSON:(NSMutableURLRequest *)request
 
     dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_HIGH, 0), ^(void) {
         NSURLSession *session = [NSURLSession sharedSession];
+        [request addValue:userAgent forHTTPHeaderField:@"User-Agent"];
```

Comment:
This needs to be passed in to the `RequestFactory` in the `TradeItRequestResponseFactory`.